docker/go-control-plane/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.22.3-bookworm
```

Comment:
The compressed size of this image is 285.45. Would it be possible to use an alpine image instead which might be significantly smaller. Doesn't *have* to be done as part of this PR. Adding a TODO would work too.